learningresources/views.py

```diff
@@ -33,6 +37,13 @@ def welcome(request):
 
 
 @login_required
+@permission_required_or_403(
+    '{}.add_{}'.format(
```

Comment:
In theory you are right, but I hate to put in the code hardcoded strings that are dependent on other code.